### PR TITLE
Added advanced configuration and code refactoring 

### PR DESCRIPTION
**Pull request by RevanthTanneeru**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [ x] You created a branch from `develop` branch.
- [ x] You PR is raised on `develop` branch and not on`main`.
- [x ] You have read the [Contribution Guidlines](CONTRIBUTING.md) before creating this PR.

### CODE REVIEW

File: cmd/add/schema/schema.go [hunk 1]
```diff
@@ -19,22 +20,6 @@ import (
...
-	Rules            string `json:"rules"`
-	Status           string `json:"status"`
-	Type             string `json:"type"`
-}
-type schemaStr struct {
-	Data []Schema `json:"Data"`
-}
-
 var urlall string
 var url1 string
```
Comment (mohammed786) [inline cmd/add/schema/schema.go:24]: We can remove this unused variables

File: cmd/add/schema/schema.go [hunk 2]
```diff
@@ -70,48 +55,67 @@ func add(temp int) error {
...
 	if err != nil {
-		return err
+		return nil
 	}
 	var temp1 []int
 	for i := 0; i < len(resultResp.Data); i++ {
 		if resultResp.Data[i].Parent == "" {
 			temp1 = append(temp1, i)
 		}
 	}
```
Comment (mohammed786) [inline cmd/add/schema/schema.go:69]: What does this means?

**Review by mohammed786 — COMMENTED** [review]
Please explain the code mentioned in the comment

File: cmd/add/schema/schema.go (hunk 1 above)
Comment (RevanthTanneeru) [reply cmd/add/schema/schema.go]: Done.

File: cmd/add/schema/schema.go (hunk 2 above)
Comment (RevanthTanneeru) [reply cmd/add/schema/schema.go:68]: We are storing the indexes of the standard fields for easier indexing.

**Review by mohammed786 — APPROVED** [review]
looks good

**Pull request merged**